crates/astria-sequencer/src/fees/query.rs

```diff
@@ -261,6 +261,7 @@ pub(crate) async fn transaction_fee_request(
         };
         fees.push((trace_denom.into(), value));
     }
+    fees.sort_by(|a: &(Denom, u128), b: &(Denom, u128)| a.0.cmp(&b.0));
```

Comment:
It looks like this change (and the addition of `PartialEq` and `Eq` to `TransactionFeeResponse`) was added to the production code purely to support testing using `assert_eq`.  If so, I'd prefer if the production code was left as is and the tests changed to support the vecs having differing orders.

Reply:
removed this line and changed the tests to sort instead.